app/src/main/kotlin/io/intrepid/skotlinton/screens/example1/Example1ViewModel.kt

```diff
@@ -0,0 +1,20 @@
+package io.intrepid.skotlinton.screens.example1
+
+import com.jakewharton.rxrelay2.PublishRelay
+import io.intrepid.skotlinton.base.BaseViewModel
+import io.intrepid.skotlinton.base.ViewModelConfiguration
+import io.intrepid.skotlinton.screens.example1.Example1ViewEvent.GotoExample2
+import io.intrepid.skotlinton.screens.example1.Example1ViewEvent.ShowCurrentTimeToast
+
+class Example1ViewModel(configuration: ViewModelConfiguration) : BaseViewModel(configuration) {
+
+    val eventPublisher: PublishRelay<Example1ViewEvent> = PublishRelay.create()
```

Comment:
I'd like to do more thinking about the use of PublishRelay here vs something else, but that can probably wait until we do more navigation stuff.

A potential alternative might be building a Relay equivalent to UnicastSubject/UnicastProcessor/MulticastProcessor. @streetsofboston built a NavigatorProcessor on a recent project that may be a good example.

Reply:
yeah that sounds good to me